installer/conf/container.conf

```diff
@@ -94,6 +94,13 @@
 	run_interval 60s
 </source>
 
+#Kubernetes logs
+<source>
+	type kubelogs
+	tag oms.api.KubeLogs
+	run_interval 5s
```

Comment:
5s run interval might be too aggressive for scale. Can you do a scale test of 1000/1000 of your poc pods logging every second ?